dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -3,7 +3,7 @@
     {%- set dest_columns = adapter.get_columns_in_relation(target_relation) -%}
     {%- set dest_cols_csv = dest_columns | map(attribute='quoted') | join(', ') -%}
     insert into {{ target_relation }}
-    select {{dest_cols_csv}} from {{ tmp_relation.include(database=false, schema=false) }};
+    select {{dest_cols_csv}} from {{ tmp_relation.include(database=true, schema=true) }};
```

Comment:
i see the following statement on Trino while executing the jaffle shop model tests

```
insert into "memory"."default_custom_schema"."dim_order_dates"
    select "order_date" from "memory"."default_custom_schema"."dim_order_dates__dbt_tmp"
```

Also
```
trino> show schemas in memory;
        Schema         
-----------------------
 default               
 default_custom_schema 
 information_schema    
(3 rows)
```

Creating new schemas is not the purpose of this PR right?

Reply:
No, the purpose is to support custom schemas if they exist. The incremental model fails if a custom schema is used.

Reply:
I had a look again at how the tests are running.

dbt is issuing this command: 

```
create schema if not exists "memory"."default_custom_schema"
```

although the incremental model is defined to run on "custom_schema"

```
{{
    config(materialized = 'incremental', schema='custom_schema')
}}
```

Please take a look again at your implementation.

BTW, pls take a look at 
https://trino.io/docs/current/installation/cli.html#examples

to setup the creation of the "custom_schema" in the Trino docker instance before starting dbt.

This can be done by simply adding a new bash script which does

```
docker container exec -t dbt-trino-trino-1  trino --execute "create schema memory.custom_schema"
```

Reply:
Creating the schema in case it doesn't exist can be done via dbt. 
However it is important that the created schema does fit the specified name.

Reply:
This is due to how dbt treats custom schemas - https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas. It basically concatenates the `<target_schema>_<custom_schema>`  by default. 

In other words, if I define a custom schema called `custom_schema` , dbt would internally translate it to `<default_schema>_<custom_schema>`

Reply:
Thanks for the heads'up @rahulj51  . 

Could you please add a README entry  about https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas ?
> ## Custom schemas
> By default, all dbt models are built in the schema specified in your target. In dbt projects with lots of models, it may be useful to instead build some models in schemas other than your target schema – this can help logically group models together.
> See [Using custom schemas](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas) for more details.

Reply:
@hovaesco will do by tomorrow.

Reply:
@hovaesco , I added the section on custom schemas to the README